Source/ThirdParty/libwebrtc/libwebrtc.xcodeproj/project.pbxproj

```diff
@@ -4538,10 +4538,11 @@
 			filePatterns = asm;
 			fileType = sourcecode.asm.asm;
 			inputFiles = (
+				"$(BUILT_PRODUCTS_DIR)/yasm",
```

Comment:
This input is added because a script phase I removed below told me to:

```
# Once support for the legacy build system is dropped, delete this build phase
# and move its input file to the "Assembly files" build rule.
```